CHALLENGE.md

```diff
@@ -52,71 +51,71 @@ The project includes:
 
 ![edit-promp](.github/edit-prompt.png)
 
-- Clicking on the `EDIT` button will open a browser prompt with a message `New Tournament Name:`, an input field with the current tournament name as pre-filled value and the buttons `Cancel` to cancel and `OK` to confirm.
+- Pressing the `EDIT` button opens a browser prompt with the prompt message `New Tournament Name:`, an input field with the current tournament name as pre-filled value and buttons `Cancel` to cancel and `OK` to confirm.
 - The tournament name must contain only Latin letters, numbers, and spaces, not an empty string or only spaces.
-- When confirming, the tournament name will be updated immediately using an optimistic update in the UI and a fetch call on the fake REST API.
+- When `OK` is selected, the tournament name updates immediately in the UI through an optimistic update and a fetch call to the fake REST API.
```

Comment:
don't we want to mention `without any loading indicators`  also here? Same principle than the delete, still optimistic update